project2/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Crowned Article Card</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xvl5rma.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xvl5rma.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xvl5rma.css">
+
+
+</head>
+
+<body>
+
+    <div class="card">
+
+        <div class="content">
+
+            <div class="img-wrapper">
+
+                <a href="#"><svg class="favorite" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 52.18">
```

Comment:
Consider moving the "favorite" class to the `a` element instead:
```suggestion
                <a href="#" class="favorite"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 52.18">
```

Since this element is absolutely positioned, you want the link and the icon to move together.